packages/teleport/src/labelClick.ts

```diff
@@ -28,6 +27,5 @@ export default function labelClick(
   const finalQuery = queryParts.join(' && ');
 
   setParams({ ...params, search: '', query: finalQuery });
-  setIsAdvancedSearch(true);
-  setSearchString(finalQuery);
+  replaceHistory(encodeUrlQueryParams(pathname, finalQuery, params.sort, true));
```

Comment:
is setting the params necessary? i thought replaceHistory will trigger a re-render since the params will change? (from the `useLocation` hook, this is a refactor you should look into since we are out of time for this one)